styles.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@700&display=swap');
+
+* {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  width: 375px;
+  display: flex;
+  height: 48px;
+  top: 44px;
+  color: #181e25;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+}
+
+.top-head {
+  display: flex;
+  width: 547.48px;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 90px;
+  height: 28px;
+  top: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+  display: flex;
+}
+
+.logotext {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  align-items: center;
+  color: #28352f;
+}
+
+.logo a {
+  text-decoration: none;
+  color: #091e42;
+  margin-left: 20px;
+}
+
+a :hover :visited {
+  color: #091e42;
+}
+
+nav {
+  display: flex;
+}
+
+.hamburger {
+  width: 18px;
+  height: 12px;
+  top: 6px;
+  margin-right: 30px;
+}
+
+.contents {
+  background-image: url(./images/header-illsutration-mobile.svg);
+  background-position: center center;
+  background-repeat: no-repeat;
+  width: 375px;
+  height: 648px;
+  top: 98.12px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  width: 375px;
+  height: 726px;
+  top: 92px;
+}
+
+.primary {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: #091e42;
+  margin: auto;
+  width: 327px;
+  top: 64;
+}
+
+.primaryblack {
+  text-align: center;
+  margin: auto;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  top: 158px;
+}
+
+.primarygreen {
+  color: #36b37e;
+}
+
+.supportingtext {
+  margin-bottom: 50px;
+  margin-top: 30px;
+}
+
+.supportingtext .para {
+  top: 58;
+  font-family: 'inter', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
+  justify-content: center;
+  color: #172b4d;
+}
+
+.socialmedia {
+  width: 170px;
+  height: 20px;
+  align-items: center;
+  justify-content: center;
+  margin: auto;
+}
+
+.ul-socialmedia {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.socialists {
+  list-style: none;
+  margin-left: 5px;
+  justify-content: center;
+  align-items: center;
+}
+
+.action {
+  justify-content: center;
+  text-align: center;
+  margin-top: 50px;
+}
+
+.act {
+  position: absolute;
```

Comment:
- [ ] [OPTIONAL] The position of this button should not be `absolute`. To center an element you can just set the parent containers display to flex and align or justify its items to center.

Reply:
Noted